source/camera/DummyArrayClass.h

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef DUMMYARRAYCLASS_H_
+#define DUMMYARRAYCLASS_H_
+
+#include "utils/Vector3i.h"
+
+#include <array>
+#include <vector>
+
+/*! \brief The DummyArrayClass class is a wrapper 
+ *  around the std::array mainly to overload the 
+ *  operator [].        
+ */
+
+
+class DummyArrayClass
```

Comment:
DummyArray sound like a bad name. Be it a wrapper with only a few changes or something that have additional features, it is the vector manager for the culling. So, CullingVectorManager makes more straight forward what this class is used for than DUmmyArrayClass.